server/resolvers/TestPlanReport/issuesResolver.js

```diff
@@ -45,6 +61,11 @@ const issuesResolver = async testPlanReport => {
                     ? 'CHANGES_REQUESTED'
                     : 'FEEDBACK';
 
+            const at = ats.find(at => at.name === hiddenIssueMetadata.atName);
+            const browser = browsers.find(
+                browser => browser.name === hiddenIssueMetadata.browserName
+            );
```

Comment:
Let's make these name checks case insensitive just in case the GitHub issue is ever edited and someone accidentally does `NVDa` instead of `NVDA`, for example.